package.json

```diff
@@ -71,6 +71,7 @@
     "lit-html": "^1.3.0",
     "node-fetch": "^2.6.0",
     "npm-run-all": "^4.1.5",
+    "octokit": "^2.0.7",
```

Comment:
I try to pin all newly introduced dependencies to have all of them pinned at one point. The reasoning behind this is that having pinned dependencies usually solves a lot of errors with lock files for not so regular contributors.